tendrl-monitoring-integration.spec

```diff
@@ -0,0 +1,53 @@
+Name: tendrl-monitoring-integration
+Version: 1.0.0
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Module for Tendrl Monitoring Integration
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/monitoring-integration
+
+#Requires: tendrl-commons
+Requires: grafana
+
+%description
+Python module for Tendrl to create a new dashboard in Grafana
+
+%prep
+%setup
+
+# Remove bundled egg-info
+rm -rf %{name}.egg-info
+
+%build
+%{__python} setup.py build
+
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/monitoring_integration
+install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring_integration
+#install -Dm 0644 monitoring-integration.service $RPM_BUILD_ROOT%{_unitdir}/monitoring-integration.service
+install -Dm 0644 tendrl/monitoring_integration/grafana/grafana.ini $RPM_BUILD_ROOT%{_sysconfdir}/grafana/grafana.ini
+install -Dm 0644 graphite.conf.sample.yml $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring_integration/graphite.conf.yml
```

Comment:
Note that these config files might be required here:

https://github.com/Tendrl/performance-monitoring/blob/master/etc/tendrl/performance-monitoring/graphite-web.conf.sample

https://github.com/Tendrl/performance-monitoring/blob/master/etc/tendrl/performance-monitoring/carbon.conf.sample

stay tuned to https://github.com/Tendrl/monitoring-integration/pull/1